internal/controller/logger_manager.go

```diff
@@ -219,6 +219,9 @@ func NewGRPClogger(grpcSyncer *BufferedGrpcWriteSyncer) *zap.SugaredLogger {
 			grpcSyncer.bufferLogEntry(&entry)
 			return nil
 		}
+
```

Comment:
Maybe add a comment here to briefly describe why your going to flush buffer every time this hook is called.

Reply:
added